### PR TITLE
cleaned up the code and fixed indentations

### PR DESCRIPTION
**Pull request by sahilasopa**
Fixed indentations and did some minor refactoring to code

### CODE REVIEW

File: app/src/main/java/io/github/danthe1st/jdoc4droid/model/textholder/HtmlStringHolder.java [hunk 1]
```diff
@@ -21,32 +21,34 @@
 
     @NonNull
     private String html;
-    @NonNull
```
Comment (danthe1st) [inline app/src/main/java/io/github/danthe1st/jdoc4droid/model/textholder/HtmlStringHolder.java:24]: If you remove `@NonNull` in here, can you please also remove `@NoArgsConstructor` and `@RequiredArgsConstructor`?

**Review by danthe1st — CHANGES_REQUESTED** [review]

**Review by danthe1st — CHANGES_REQUESTED** [review]
It seems like you forgot to reformat `io.github.danthe1st.jdoc4droid.activities.list` and `io.github.danthe1st.jdoc4droid.activities.list.javadocs`.

**Comment by sahilasopa** [comment]
> It seems like you forgot to reformat `io.github.danthe1st.jdoc4droid.activities.list` and `io.github.danthe1st.jdoc4droid.activities.list.javadocs`.

alright done

File: app/src/main/java/io/github/danthe1st/jdoc4droid/activities/list/javadocs/ListJavadocsActivity.java [hunk 2]
```diff
@@ -51,12 +51,37 @@
 
     private ProgressBar progressBar;
 
+    @WorkerThread
```
Comment (danthe1st) [inline app/src/main/java/io/github/danthe1st/jdoc4droid/activities/list/javadocs/ListJavadocsActivity.java]: Again, please don't move methods.

**Review by danthe1st — CHANGES_REQUESTED** [review]

**Review by danthe1st — APPROVED** [review]

**Pull request merged**